protocols/luci-proto-ipv6/htdocs/luci-static/resources/protocol/dhcpv6.js

```diff
@@ -30,6 +30,10 @@ return network.registerProtocol('dhcpv6', {
 		o.value('64');
 		o.default = 'auto';
 
+		o = s.taboption('general', form.Flag, 'norelease', _('Do not send a Release when restarting'),
+						_('Enable to reduce the risk of prefix changes following a restart'));
+		o.default = '1';
```

Comment:
The uci option defaults to off, not on. Please remove the `o.default = 1` line.

Reply:
Changing the default was the intent of this PR (as well as expose the config var via Luci), given the new advice from RFC 9096.    Not sure where to change the uci option default, but if you point me in the right direction I could create another PR there.  Presumably starting a _"should we change this default?"_ discussion officially.

Reply:
The option default would need to be changed here: https://github.com/openwrt/openwrt/blob/main/package/network/ipv6/odhcp6c/files/dhcpv6.sh

Reply:
Is that by changing [Line18](https://github.com/openwrt/openwrt/blob/main/package/network/ipv6/odhcp6c/files/dhcpv6.sh#L18) from

`proto_config_add_string 'norelease:bool'`
to
`proto_config_add_string 'norelease:bool:1'`

correct?